src/shared/Microsoft.Git.CredentialManager/Authentication/WindowsIntegratedAuthentication.cs

```diff
@@ -14,6 +14,12 @@ public interface IWindowsIntegratedAuthentication : IDisposable
 
     public class WindowsIntegratedAuthentication : IWindowsIntegratedAuthentication
     {
+        public static readonly string[] AuthorityIds =
+        {
+            "Integrated", "Windows", "Kerberos", "NTLM",
```

Comment:
I'm confused what some of these AuthorityIds are.  How did you get this list?

Reply:
These values came from Windows GCM:

https://github.com/microsoft/Git-Credential-Manager-for-Windows/blob/9270c89a0ce109c7a86e9b3d8174a0af56aab468/Shared/Cli/Functions/Common.cs#L433-L474